### PR TITLE
fix: Implement dataset exception uint test [#42]

### LINKED ISSUE #42: implement dataset exception uint test
refer to: https://github.com/dataswap/core/issues/60
Complete Exception Test Cases

### PR DESCRIPTION
**Pull request by lovel8**
## Related Issues

close #42 

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: test/v0.8/uinttests/module/dataset/Approve.t.sol [hunk 1]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {ApproveTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/ApproveTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/ApproveTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/Approve.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/ApproveMetadata.t.sol [hunk 2]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {ApproveMetadataTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/ApproveMetadataTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/ApproveMetadataTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/ApproveMetadata.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/Reject.t.sol [hunk 3]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {RejectTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/RejectTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/RejectTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/Reject.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/RejectMetadata.t.sol [hunk 4]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {RejectMetadataTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/RejectMetadataTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/RejectMetadataTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/RejectMetadata.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/SubmitMetadata.t.sol [hunk 5]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {SubmitMetadataTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/SubmitMetadataTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/SubmitMetadataTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/SubmitMetadata.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/SubmitProof.t.sol [hunk 6]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {SubmitProofTestCaseWithSuccess} from "test/v0.8/testcases/module/dataset/SubmitProofTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/SubmitProofTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/SubmitProof.t.sol:20]: Change the import format.

File: test/v0.8/uinttests/module/dataset/SubmitVerification.t.sol [hunk 7]
```diff
@@ -17,7 +17,7 @@
 pragma solidity ^0.8.21;
 
 import {Test} from "forge-std/Test.sol";
-import {SubmittVerificationTestCaseWithSuccess, SubmittVerificationTestCaseWithFail} from "test/v0.8/testcases/module/dataset/SubmittVerificationTestSuite.sol";
+import "test/v0.8/testcases/module/dataset/SubmittVerificationTestSuite.sol";
```
Comment (siriusyim) [inline test/v0.8/uinttests/module/dataset/SubmitVerification.t.sol:20]: Change the import format.

**Review by siriusyim — CHANGES_REQUESTED** [review]

**Comment by lovel8** [comment]
@siriusyim Have done

**Review by siriusyim — APPROVED** [review]

**Review by waynewyang — APPROVED** [review]

**Pull request merged**